### PR TITLE
RDISCROWD-2234 Project page required fields asterisk

### PR DESCRIPTION
**Pull request by kbecker42**
- Replaced red border with asterisk for required fields. 
- Added `required=true` to the `render_field()` macro to indicate required fields.

## Example

`{{ render_field(form.name, class_="span4", placeholder=_('The name of the project'), required=true) }}`

## Screenshot

cap

### CODE REVIEW

File: static/sass/custom/project.scss [hunk 1]
```diff
@@ -28,4 +22,10 @@
       font-style: italic;
       margin: 5px 5px 10px 5px;
     }
+
```
Comment (mariarivera30) [inline static/sass/custom/project.scss]: Can we make this global? I think we probably could add this require on other forms eventually.

File: templates/_formhelpers.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
-{% macro render_field(field, label_visible=true) -%}
-     <div class="form-group {% if field.errors %}has-error{% endif %} {{ kwargs.pop('class_', '') }}">
+{% macro render_field(field, label_visible=true, required=false) -%}
```
Comment (mariarivera30) [inline templates/_formhelpers.html]: I don't think you need to pass a flag, each field has a flag for required already. `field.flags.required`

Comment (kbecker42) [reply templates/_formhelpers.html]: Thank you, great find! https://github.com/bloomberg/pybossa-default-theme/pull/214/commits/8954cc49372345b47b7066176624317048914086

File: static/sass/custom/project.scss (hunk 1 above)
Comment (kbecker42) [reply static/sass/custom/project.scss]: Yes, agreed. :) https://github.com/bloomberg/pybossa-default-theme/pull/214/commits/8954cc49372345b47b7066176624317048914086

**Pull request merged**